tests/net/icmpv6/src/main.c

```diff
@@ -102,6 +102,7 @@ void test_icmpv6(void)
 
 	pkt = net_pkt_alloc_with_buffer(NULL, ICMPV6_MSG_SIZE,
 					AF_UNSPEC, 0, K_SECONDS(1));
+	zassert_not_null(pkt, "Allocation failed");
```

Comment:
Does this allow you to suppress the Coverity issue? Maybe just return if pkt == NULL?

Reply:
in unit tests we need to use ztest lib to know where test failed and why.

Reply:
Was just concerned, @tbursztyka that the Coverity tool will report this bug again.
The zassert_not_null certainly fixes the actuall issue here.